### PR TITLE
MM-11623: Add GPG signatures to version archive (v4.10+)

### PR DESCRIPTION
**Pull request by jasonblais**
@crspeller Is a key needed for verifying the GPG signature? My guess is 'yes' but wanted to confirm first.

https://mattermost.atlassian.net/browse/MM-11623

### CODE REVIEW

File: source/administration/version-archive.rst [hunk 1]
```diff
@@ -1,27 +1,35 @@
 Version Archive
 ===========================
 
+To verify the GPG signature of a Mattermost release, use the public key stored in Keybase: https://keybase.io/mattermost.
```
Comment (jasonblais) [inline source/administration/version-archive.rst]: @crspeller Added a link to keybase at top of the base.

Comment (crspeller) [reply source/administration/version-archive.rst]: Looks good.

Comment (wiersgallak) [inline source/administration/version-archive.rst]: Do customers understand why they would verify the gpg signature?  Would it be helpful to provide a little more information on how this is beneficial to them? 0/5-just a thought as I read through this.

Comment (jasonblais) [reply source/administration/version-archive.rst]: @wiersgallak Good suggestion. How would this sound?

```
If you want to check that the version of Mattermost you are installing is the official, unmodified version, compare the SHA-265 checksum or the file's GPG signature with the one published in this version archive. To verify the GPG signature of a Mattermost release, use the public key stored in Keybase: https://keybase.io/mattermost.
```

Comment (wiersgallak) [reply source/administration/version-archive.rst]: Looks great!

**Pull request merged**